.github/actions/clp-execution-image-build/action.yaml

```diff
@@ -0,0 +1,57 @@
+name: "clp-execution-image-build"
+description: "Builds a container image that contains the dependencies necessary
+to run the CLP package."
+
+inputs:
+  containerImageRegistry:
+    default: "ghcr.io"
+    description: "Container Image Registry"
+    required: false
+  containerImageRegistryUsername:
+    default: "${{github.actor}}"
+    description: "Container Image Registry Username"
+    required: false
+  containerImageRegistryPassword:
+    default: ""
+    description: "Container Image Registry Password"
+    required: false
+  platformId:
+    description: "Platform ID of the container. e.g., ubuntu."
+    required: true
+  platformVersionId:
+    description: "Platform VERSION_ID / VERSION_CODENAME of the container.
+      e.g., jammy, focal."
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: "docker/login-action@v3"
```

Comment:
Docker [recommends](https://github.com/marketplace/actions/build-and-push-docker-images#usage) using the `docker/setup-buildx-action`. I know it wasn't in the original workflow, but perhaps we can add it now.